cabal.project

```diff
@@ -54,8 +54,9 @@ source-repository-package
   tag: d93afd2dc78c2f6910766fabc2f5e4452bcae2f2
   subdir: x509-system
 
-index-state: hackage.haskell.org 2022-02-02T15:17:53Z
+index-state: hackage.haskell.org 2022-04-13T23:32:58Z
 
 package *
   extra-include-dirs: /opt/homebrew/include
-  extra-lib-dirs: /opt/homebrew/lib
\ No newline at end of file
+  extra-lib-dirs: /opt/homebrew/lib
```

Comment:
Seems weird to have absolute paths in our `cabal.project`.  They won't exist on all systems.  Shouldn't these be in a `macos` specific version?

Reply:
They're ignored if they aren't present.  This is also something we can fix later, it just happened to be relevant to get @kitified to be able to build GHC 9 on her machine.  I expect someone else to run into this, but I think we can fix it reactively instead of proactively without too much trouble.